Unable to run renovate-config-validator in GitHub Actions

Ref: https://github.com/vrc-plugin/send_vrc_extension/actions/runs/4861796127/jobs/8667281944?pr=207

```log
Run renovate-config-validator
  renovate-config-validator
  shell: /usr/bin/bash -e {0}
 INFO: Validating renovate.json
 WARN: renovate.json is not valid Renovate config
       "err": {
         "message": "structuredClone is not defined",
         "stack": "ReferenceError: structuredClone is not defined\n    at massageConfig (/opt/hostedtoolcache/node/16.14.0/x64/lib/node_modules/renovate/lib/config/massage.ts:19:[2](https://github.com/vrc-plugin/send_vrc_extension/actions/runs/4861796127/jobs/8667281944?pr=207#step:5:2)6)\n    at validate (/opt/hostedtoolcache/node/16.14.0/x64/lib/node_modules/renovate/lib/config-validator.ts:[3](https://github.com/vrc-plugin/send_vrc_extension/actions/runs/4861796127/jobs/8667281944?pr=207#step:5:3)[4](https://github.com/vrc-plugin/send_vrc_extension/actions/runs/4861796127/jobs/8667281944?pr=207#step:5:5):39)\n    at /opt/hostedtoolcache/node/1[6](https://github.com/vrc-plugin/send_vrc_extension/actions/runs/4861796127/jobs/8667281944?pr=207#step:5:7).14.0/x64/lib/node_modules/renovate/lib/config-validator.ts:84:1[7](https://github.com/vrc-plugin/send_vrc_extension/actions/runs/4861796127/jobs/8667281944?pr=207#step:5:8)"
       }
Error: Process completed with exit code 1.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 16.14.0
+          node-version: 18.16.0
           cache: npm
       - run: npm ci
       - run: npm run build

--- a/.github/workflows/format.yml
+++ b/.github/workflows/format.yml
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 16.14.0
+          node-version: 18.16.0
           cache: npm
       - run: npm ci
       - run: npm run format:check

--- a/.github/workflows/lint.yml
+++ b/.github/workflows/lint.yml
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 16.14.0
+          node-version: 18.16.0
           cache: npm
       - run: npm ci
       - run: npm run lint

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -15,7 +15,7 @@ jobs:
       - name: Build
         uses: actions/setup-node@v3
         with:
-          node-version: 16.14.0
+          node-version: 18.16.0
           cache: npm
       - run: npm ci
       - run: npm run build

--- a/.github/workflows/renovate-config-validator.yml
+++ b/.github/workflows/renovate-config-validator.yml
@@ -16,7 +16,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: 16.14.0
+          node-version: 18.16.0
           cache: npm
 
       - name: Install Renovate CLI

